asdf/asdf.py

```diff
@@ -119,6 +120,13 @@ def __init__(
         else:
             self._version = versioning.AsdfVersion(validate_version(version))
 
+        if extensions is not None:
+            warnings.warn(
+                "The 'extensions' argument to AsdfFile is deprecated.  Use asdf.get_config().add_extension to "
+                "add an extension at runtime.",
+                AsdfDeprecationWarning,
```

Comment:
Similar to the other suggestion:
```suggestion
                "The 'extensions' argument to AsdfFile is deprecated.  Please see :ref:`extending_extensions_installing_asdf_config`.",
                AsdfDeprecationWarning,
```